app/src/main/java/org/listenbrainz/android/viewmodel/DashBoardViewModel.kt

```diff
@@ -71,6 +72,22 @@ class DashBoardViewModel @Inject constructor(
         }
     }
 
+    object CrashReporter {
```

Comment:
Here's how sentry's functions should be used in this class:

1. Create an interface named `SentryController` in another file.
2. Implement the above interface in that file itself and inject it via hilt into this view-model.
3. Use the injected instance to enable or disable sentry sending events.